### PR TITLE
Meet Your New Dungeon Group Members in Seconds!

### PR DESCRIPTION
**Pull request by hannibal002**
image

This feature allows players to quickly view the profile of another player who has joined their dungeon group through the party finder by hovering and clicking on the chat message announcing the player's arrival. This is a useful feature for several reasons.

First, it allows players to get to know their new group members better before embarking on their dungeon adventure together. For example, if a player is a tank, they may want to know more about the healer's gear and skills to ensure that they are well-equipped to support the group. Similarly, a player who is new to the game may want to learn more about their group members' achievements and experience to get a sense of their expertise.

Second, the profile viewer provides detailed information about the player's character, including their gear, skills, and achievements. This is useful for players who want to optimize their group composition and strategize for their dungeon run. For example, if a player is looking for a specific type of gear or skill, they can use the profile viewer to find a group member who meets those requirements.

Third, this feature helps foster a sense of community and collaboration within the game. By allowing players to connect with and learn more about their fellow adventurers, it encourages teamwork and cooperation during dungeon runs. This can lead to a more enjoyable and successful gaming experience for all group members.

Overall, the ability to quickly view the profile of another player who has joined their dungeon group is a valuable and convenient feature that enhances the gameplay experience for players.

### CODE REVIEW

**Comment by NopoTheGamer** [comment]
#367 :trollface:

**Comment by hannibal002** [comment]
Why is this still in draft??

File: src/main/java/io/github/moulberry/notenoughupdates/listener/ChatListener.java [hunk 1]
```diff
@@ -55,12 +55,15 @@
 import static io.github.moulberry.notenoughupdates.overlays.SlayerOverlay.timeSinceLastBoss2;
 
 public class ChatListener {
+
 	private final NotEnoughUpdates neu;
-	private static final Pattern SLAYER_XP = Pattern.compile(
+
+	private static final Pattern slayerExpPattern = Pattern.compile(
```
Comment (NopoTheGamer) [inline src/main/java/io/github/moulberry/notenoughupdates/listener/ChatListener.java:61]: why did you remove the capitals? The names of variables declared class constants and of ANSI constants should be all uppercase with words separated by underscores ("_").

Comment (hannibal002) [reply src/main/java/io/github/moulberry/notenoughupdates/listener/ChatListener.java:61]: Crystal Hollows Overlay and Custom AH classes uses the other format too. so we change them everywhere ig

**Review by romangraef — APPROVED** [review]
Approving for now, but i am with nopo on the convention for constants.

**Review by romangraef — APPROVED** [review]

**Review by NopoTheGamer — APPROVED** [review]

**Review by Lulonaut — APPROVED** [review]
rest in peace #367
